pkg/validations/cluster.go

```diff
@@ -79,3 +79,26 @@ func ValidateK8s126Support(clusterSpec *cluster.Spec) error {
 	}
 	return nil
 }
+
+// ValidateManagementClusterBundlesVersion checks if management cluster's bundle version is greater than or equal to the bundle version used to upgrade a workload cluster.
+func ValidateManagementClusterBundlesVersion(ctx context.Context, k KubectlClient, mgmtCluster *types.Cluster, clusterSpec *cluster.Spec) error {
```

Comment:
Would you mind clarifying the cluster spec is for the workload cluster -  perhaps with a variable change (e.g. `workload`)?